docs/appendices.txt

```diff
@@ -8,6 +8,69 @@
 Refer to the JanusGraph https://javadoc.io/doc/org.janusgraph/janusgraph-core/$MAVEN{project.version}[API documentation] for a complete documentation of all core APIs exposed by JanusGraph.
 We *strongly* encourage all users of JanusGraph to use the Gremlin query language for any queries executed on JanusGraph and to not use JanusGraph's APIs outside of the management system.
 
+[[escustomauth]]
+[appendix]
+
+== Elasticsearch Custom Authenticator Example
```

Comment:
I think you could put this in the Elasticsearch section instead of the appendix.

Reply:
I have no preferences - I thought that this sort of stuff looks better in an appendix. Moving.